source/MissionPanel.cpp

```diff
@@ -352,14 +353,18 @@ bool MissionPanel::Click(int x, int y, int clicks)
 		unsigned index = max(0, (y + static_cast<int>(availableScroll) - 36 - Screen::Top()) / 20);
 		if(index < available.size())
 		{
+			const auto lastAvailableIt = availableIt;
 			availableIt = available.begin();
 			while(index--)
 				++availableIt;
 			acceptedIt = accepted.end();
 			dragSide = -1;
-			selectedSystem = availableIt->Destination()->GetSystem();
-			DoScroll(available, availableIt, availableScroll, false);
-			CenterOnSystem(selectedSystem);
+			if(lastAvailableIt == availableIt)
```

Comment:
Not super important, and maybe someone else agrees, but since `lastAvailableIt` is the constant and `availableIt` is what you want to check is the same as it, can the order of these be reversed? Same goes for the similar line with `acceptedIt` later.